### ISSUE TITLE
Alt text requirement not documented clearly

### ISSUE THREAD

**Issue by autay27** [11ty/eleventy-img#299]
As a beginner using Eleventy I spent a long time trying to debug why my images were being rendered to the site with the wrong paths, failing to realise that they would render correctly if I added alt text.

Afterwards I noticed that it is mentioned only way down in the [description for htmlOptions](https://www.11ty.dev/docs/plugins/image/#return-type-and-html-options) as a single line commented '// required'.

Have I understood the issue correctly? If so, how can the requirement be documented more clearly? I also think an error or warning message should be printed to notify the user when the alt text is missing and why this leads to a broken image.
